HHS/Head-Start-TTADP
# [TTAHUB-44] add foreign key and some predicate indices

## Description of change

Adds indices in places where it's most likely to help database performance more than it hurts it.

## How to test

- deploy and watch the indexes get created
- make sure there are no unexplained slowdowns when writing new data. adding indexes adds a small amount of overhead to database write operations that shouldn't be noticeable, but could be in combination with other issues.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-44


## Checklists



### Every PR



### Production Deploy


### After merge/deploy
